src/main/java/com/example/demo/Exceptions/Advices/MovieNotFoundAdvice.java

```diff
@@ -0,0 +1,19 @@
+package com.example.demo.Exceptions.Advices;
+
+import com.example.demo.Exceptions.MovieNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
```

Comment:
There is an **_@RestControllerAdvice_** annotantion, could you review what are the differences between them and define which one you should use please?